### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by JakubOnderka**
@stefanides

### CODE REVIEW

File: backend/docker-compose.yml [hunk 1]
```diff
@@ -1,7 +1,52 @@
-version: '3'volumes:
-
+version: '3.4'
```
Comment (quentincaffeino) [inline backend/docker-compose.yml:1]: Specifying version as of 1.27.0 (2020-09-07) is not required

**Review by quentincaffeino — COMMENTED** [review]
If these docker-compose files are used in production dropping non required capabilities will increase security.

https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html/container_security_guide/linux_capabilities_and_seccomp

File: backend/docker-compose.yml (hunk 1 above)
Comment (JakubOnderka) [reply backend/docker-compose.yml]: Thanks, fixed

File: backend/docker-compose.yml [hunk 2]
```diff
@@ -1,7 +1,52 @@
...
 
 services:
+  php:
```
Comment (karmi) [inline backend/docker-compose.yml:2]: Nebylo by lepší tohle nazvat `app`?

Comment (JakubOnderka) [reply backend/docker-compose.yml:2]: Vycházel jsem z originálního docker-compose z https://github.com/dunglas/symfony-docker/blob/main/docker-compose.yml. Ale klidně to můžeme upravit.

**Pull request merged**